packages/Radio/src/radio.scss

```diff
@@ -1,115 +1,161 @@
+    --ids-radio-font-weight: #{tokens.$font-weight-regular};
+    --ids-radio-font-size: #{tokens.$font-size-4};
+    --ids-radio-font-size-small: #{tokens.$font-size-3};
+    --ids-radio-font-size-description: #{tokens.$font-size-3};
+    --ids-radio-text: #{tokens.$grey-800};
+    --ids-radio-text-description: #{tokens.$grey-600};
+    --ids-radio-description-padding: #{tokens.$space-1};
+    --ids-radio-padding: #{tokens.$space-2};
+    --ids-radio-margin: .2rem 0 0;
+    --ids-radio-size: #{tokens.$font-size-4};
+    --ids-radio-check: #{tokens.$samoyed};
+    --ids-radio-border: #{tokens.$grey-400};
+    --ids-radio-border-hover: #{tokens.$grey-800};
+    --ids-radio-border-disabled: #{tokens.$grey-300};
+    --ids-radio-border-checked: #{tokens.$electric-blue-500};
+    --ids-radio-background: #{tokens.$samoyed};
+    --ids-radio-background-disabled: #{tokens.$grey-200};
+    --ids-radio-background-checked: #{tokens.$electric-blue-500};
+    --ids-radio-background-disabled-checked: #{tokens.$grey-300};
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-radio-font-family: var(--hop-body-md-font-family);
+    --ids-radio-font-weight: var(--hop-body-md-font-weight);
+    --ids-radio-font-size: var(--hop-body-md-font-size);
+    --ids-radio-font-size-small: var(--hop-sm-font-size);
+    --ids-radio-font-size-description: var(--hop-body-sm-font-size);
+    --ids-radio-text: var(--hop-neutral-text);
+    --ids-radio-text-description: var(--hop-neutral-text-weak);
+    --ids-radio-description-padding: 0;
+    --ids-radio-padding: var(--hop-space-inline-sm);
+    --ids-radio-margin: .125rem 0 0;
+    --ids-radio-size: 1rem;
+    --ids-radio-check: var(--hop-neutral-surface);
+    --ids-radio-check-size: .5rem;
+    --ids-radio-check-background-disabled: var(--hop-neutral-icon-disabled);
+    --ids-radio-border: var(--hop-neutral-border);
+    --ids-radio-border-hover: var(--hop-neutral-border-hover);
+    --ids-radio-border-disabled: var(--hop-neutral-border-disabled);
+    --ids-radio-border-checked: var(--hop-neutral-border-active);
+    --ids-radio-border-checked-hover: var(--hop-neutral-border-active);
+    --ids-radio-background: var(--hop-neutral-surface);
+    --ids-radio-background-hover: var(--hop-neutral-surface-hover);
+    --ids-radio-background-disabled: var(--hop-neutral-surface-disabled);
+    --ids-radio-background-disabled-checked: var(--hop-neutral-surface-disabled);
+    --ids-radio-background-checked: var(--hop-neutral-surface-active);
+    --ids-radio-check-background-checked-hover: var(--hop-neutral-icon-hover)
 }
 
 %radio {
-  content: '';
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  width: 0.6rem;
-  height: 0.6rem;
-  border-radius: 50%;
-  background: var(--ids-radio-background);
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: var(--ids-radio-check-size, .6rem);
```

Comment:
.6rem in Igloo != to .6rem in Workleap this probably should be 0.375rem in Workleap.

Reply:
In fact, the value here is used as a callback because the `--ids-radio-check-size` variable doesn't exist in Igloo. This approach was used to avoid adding variables to Igloo.